conf/application.conf

```diff
@@ -86,6 +86,10 @@ microservice {
       host = localhost
       port = 8500
     }
+    
+    auth-login-api {
+      url = "http://localhost:8585"
+    }
 
```

Comment:
did this need to be feature switched so that it only exists in external test?

Reply:
No. It's testOnly, so it's not available in production. That's all that matters.